bitnami/superset/templates/flower/deployment.yaml

```diff
@@ -0,0 +1,189 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.flower.enabled }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ printf "%s-flower" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
```

Comment:
I'd create a "superset.flower.fullname" helper for this (and for other components) so you avoid this piece of code on several places.